one-ui/src/main/java/com/marklogic/hub/curation/services/FlowManagerService.java

```diff
@@ -38,6 +38,10 @@
     private Scaffolding scaffolding;
 
     public Flow updateFlow(String flowJson) {
+        return updateFlow(flowJson, false);
```

Comment:
No action needed in this PR, but the next time we have to touch this class, let's fold it into FlowManager.